app/controllers/candidate_interface/apply_controller.rb

```diff
@@ -0,0 +1,8 @@
+module CandidateInterface
+  class ApplyController < CandidateInterfaceController
+    def show
+      @provider_code = params[:providerCode]
```

Comment:
If we use `params.fetch(:providerCode)` Rails will raise a specific error that we can catch if the arguments are missing:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/2fd2131dbf9d51c0f6d1a2f05571f53992ff0b3a/app/controllers/vendor_api/applications_controller.rb#L3